cloud: scale up/down gcp dedicated nodes (#13440)
pingcap/docs: tidb-cloud/scale-tidb-cluster.md

> **Note:**
>

> > - Changing node size is only available to clusters that are:
> >     - hosted on AWS and created after 2022/12/31.
> >     - hosted on GCP and created after 2023/04/26.

Reviewer: ```suggestion
> - Changing node size is only available to the following clusters:
>     - hosted on AWS and created after 2022/12/31.
>     - hosted on GCP and created after 2023/04/26.
```

Reviewer: When a colon introduces a list, the text that precedes the colon must be able to stand alone as a complete sentence.
See https://developers.google.cn/style/colons#introductory-phrase-preceding-colon
